src/query/storages/parquet/src/parquet_table/table.rs

```diff
@@ -149,9 +143,9 @@ pub(crate) fn arrow_to_table_schema(mut schema: ArrowSchema) -> TableSchema {
     TableSchema::from(&schema)
 }
 
-pub(super) fn create_parquet_table_info(table_id: u64, schema: ArrowSchema) -> TableInfo {
+pub(super) fn create_parquet_table_info(schema: ArrowSchema) -> TableInfo {
     TableInfo {
-        ident: TableIdent::new(table_id, 0),
+        ident: TableIdent::new(0, 0),
```

Comment:
If it's safe to let the table id be 0? cc @sundy-li

Reply:
id of table_function and table in meta overlap too.
seem only useful for table in meta.